steps/server10.js

```diff
@@ -0,0 +1,28 @@
+// Now the socket io is applied to our server, everything is configured
```

Comment:
I think there might be a step missing here, but no matter. `io.on("connection"...` could be broken up into two small steps to break up this good explanation:

```
// Every Socket.IO interaction begins with the socket connection from the client to the server.
// To listen to the connection event we have to call the `on` method from the `io` instance we created.
// Chain `on` to `io` and pass it "connection" to listen for any incoming connection events.
```

The next step could read:
```
// Pass a callback function as the second parameter to `on` and have it log "User connected" to the console whenever the connection event is fired.
```